app/views/doorkeeper_applications/access_logs.html.erb

```diff
@@ -0,0 +1,42 @@
+<% content_for :title, "#{@application.name} access log" %>
+
+<% content_for :breadcrumbs,
+               render("govuk_publishing_components/components/breadcrumbs", {
+                 collapse_on_mobile: true,
+                 breadcrumbs: [
+                   {
+                     title: "Dashboard",
+                     url: root_path,
+                   },
+                   {
+                     title: "Applications",
+                     url: doorkeeper_applications_path,
+                   },
+                   {
+                     title: @application.name,
+                     url: edit_doorkeeper_application_path(@application),
+                   }
+                 ]
+               })
+%>
+
+<% if @logs.any? %>
```

Comment:
It might be useful to provide a bit of an explanation of this data for users. Particularly the fact that this data only goes back 2 years would be a useful caveat to be aware of.

Reply:
Added in 168dd3f